pandas/tests/window/test_numba.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+import pytest
+
+import pandas.util._test_decorators as td
+
+from pandas import Series
+import pandas.util.testing as tm
+
+
+@td.skip_if_no("numba", "0.46.0")
```

Comment:
Can just keep as `skip_if_no("numba")` since the minimum is already enforced by requirements files